_faqs/04-01-add-a-project.markdown

```diff
@@ -0,0 +1,14 @@
+---
+question: How do I add a project to the community project page on opensearch.org?
+category: Community and Collaboration
+---
+
+To add a project:
+
+* Fork the [opensearch website repository](https://github.com/opensearch-project/project-website).
+* Create a branch. I recommend naming it `adds-<your project name>-to-community-projects`
```

Comment:
Suggest updating this to say "The recommended naming convention is `adds-<your project name>...."